### ISSUE TITLE
Project/Dev Progress

### ISSUE THREAD

**Issue by einthusan** [Kee2u/Deciphering_the_Indus_Valley_Script#42]
Hello,

I was wondering if there was any progress on this project? I would be interested in helping with this.

Thanks,
Einthusan

**Comment by jjasim** [comment]
Hello,

I'd be interested too, especially in preparing a properly labelled dataset of the Sangam works to represent Old Tamil, for more robust analysis. 

Thanks

**Comment by einthusan** [comment]
What I noticed is that Mahadevan assumes the "an" as the most frequent terminator in Indus scripts because the Sangam literature represented a patriarchal culture whereas Indus Valley was matriarchal. However, Wells was I believe more accurate in claiming it was "ay/ai" as in Cow or Mother in Old Tamil. I believe it may have been actually mother and later day Vedic schools had brought the cow worship cult which doesn't seem so prevalent in Indus culture as all animals were treated similarly (even the unicorn was probably just a dominant clan symbol which shows no continuity to later day Vedic cow worship cult). Anyhow, my point being that Sangam works do have Old Tamil words but the context is remote to Indus as culture had shifted from matriarchy to patriarchy. Closest and oldest useful works would be Ainthiram (Tamil) which is often never talked about but Tolkkappiyar is said to have been well versed in it according to the preface of Tolkkappiyam. I have the PDF and the English translation which was based on a Tamil translation was totally wrong in my opinion because the translators did not have a grounded context of the materialistic non-vedic nature of the Old Tamil world. We should start at properly translating Ainthiram to have any chance at decipherment. Would like to hear your opinion.

**Comment by jjasim** [comment]
Hi so sorry for the late reply, I missed the notification for this.

Your idea does seem interesting and make sense. But does the Ainthiram survive to today? I thought it was a lost work!
If you have a copy of it, please do share, I would love to take a look. Do you know when the text is dated to?

**Comment by einthusan** [comment]
Hey,

Sorry for my late response as well. Been a little busy. Below I have attached some useful things, starting with my wife and I giving an initial try at a proper English translation of the works which never went that far but was enlightening for us. Then there is a download link that will expire in 7 days, let me know if u need me to re-upload it, but it has 2 books. The Tamil re-print in 1986 and the English translation version printed in 1997 or so. Ignore all the preface and introductory stuff, and get to the actual verses which are numbered, as I have shown in the attached sample picture. Let me know how I can help.

Edit: I totally disagree with the English translation of 1997, and I feel it is so useless that it shouldn't even be considered at all.

Cheers,
Einthusan

Sample translation by us
[Aintiram Translation.docx](https://github.com/Kee2u/Deciphering_the_Indus_Valley_Script/files/12446745/Aintiram.Translation.docx)

Ainthiram Books (Tamil and English w/Translations)
https://we.tl/t-7IvFw99vAv

Example:
<img width="778" alt="Screen Shot 2023-08-26 at 5 27 28 PM" src="https://github.com/Kee2u/Deciphering_the_Indus_Valley_Script/assets/1294473/d04d6d79-1ef8-4133-8145-d027c33e9505">

**Comment by Kee2u** [comment]
Hi there,

Thank you so much for your interest. I started this project to switch careers from Mechanical Engineering to Data Engineering but once I did, I put this on the backburner.
This looks promising, would you two be available for a call sometime? I wonder if we could use LLMs to help with the decipherment too.

**Comment by einthusan** [comment]
@Kee2u Yes, I would be available for a call, and this is something of my passion. I could afford to help fund this a bit too, to help with any associated costs to move this forwards. I am in Toronto time, just fyi. My email is just my first name @ gmail.com. I will be available starting Sept 1.

Cheers,
Einthusan

**Comment by aravindarc** [comment]
I am also interested in this work. Since Indus script is considered largely to be Logograms, I thought of applying Tholkappiam's Soladhigaram (mainly the punarchi vidhigal) to Indus Logograms, to find if they abide by a similar grammar( and if they do, identify nouns and verbs). I wasn't aware of Ainthiram. 

@Kee2u How do I get access to the Indus Script corpus, I found this webapp https://indusscript.in/, which seems to have concordance also, but it couldn't be crawled as it is rendered as a canvas. I want to do the same thing that you've done, load the indus logograms into a Vector database and attach all metadata as vectors. Any help here would be appreciated.

@einthusan I would be happy to help you in translating Ainthiram, and could you also share the Tamil version.

**Comment by jjasim** [comment]
@aravindarc 
About the https://indusscript.in/, yeah that's a great pity as I really wanted to compile a dataset of all the Indus inscriptions and the other associated information to each inscription as a dataset (if you are aware of such a dataset, I'd be happy to know). Having a comprehensive and useable dataset that is available to all is key for more quality decipherment efforts on the script IMO. I'd be happy to help if you have any ideas for compiling the https://indusscript.in/ or some other source into a nice dataset.

As for your logogram idea, you might be interested in this reddit post I came across a while ago: https://www.reddit.com/r/Dravidiology/comments/15n60zd/some_visualisations_of_acai_split_sangam_era_poet/

**Comment by aravindarc** [comment]
@jjasim 
I have written to RMRL, to see if they can provide access to the database. I am waiting for their reply. I will update this thread If I get my hands on something. We can look for acquiring other datasets after that, if RMRL refuses.

**Comment by einthusan** [comment]
@aravindarc @jjasim Please see download link. It expires in 7 days because of WeTransfer, which is where I uploaded it due to 25MB file size limit on Github. Below is an image showing what files can be found inside.

https://we.tl/t-bqyH2utHbx

<img width="410" alt="Tamil Books Indus" src="https://github.com/Kee2u/Deciphering_the_Indus_Valley_Script/assets/1294473/5f5f0424-2de5-4a2f-b8a6-efbb0f8504d8">
 
I am ready to help fund this work. I am really interested in translating Ainthiram first, then using that for further decipherment. I am available to meet up remotely or in person. Do let me know what you all think.

PS: Ainthiram starts at page 30, and Tolkkapiam starts at page 23. I have added Lokayata book by Debiprasad, which I believe contains the culture for Indus Valley. I haven't seen any sources out there that make this connection, but I strongly believe the philosophy of the Indus Valley should be of Lokayatham (Carvakam).

**Comment by Yashwanth-Chandrakumar** [comment]
Hi Guys can we all work together to finding an solution

**Comment by einthusan** [comment]
I'm still down to help. Anyone have a plan?

**Comment by Yashwanth-Chandrakumar** [comment]
Hi @einthusan it would be good to go with the Dravidian Hypothesis but we need datasets of ancient scripts that have been finely translated. Anyone know anybody who is related to Archaeology?
I have ideas on making this opensource. What is your opinion

**Comment by nielsle** [comment]
Bahati Asumali  has published a ipython notebook that she used  to load data from Mahadevan's digital corpus of Indus inscriptions. See  the supplementary information section of Bahati Asumali Palgrave Commun 5, 73 (2019)

https://www.nature.com/articles/s41599-019-0274-1#additional-information

I don't think that it is opensource, but perhaps it can serve as a inspiration.

**Comment by einthusan** [comment]
No connections and I personally believe it's proto-Tamil .. it's also a matriarchal society, non-Vedic, and closely matches that of Lokayata/Carvaka philosophy. See good book by Debbiprashad called "Lokayata" ... https://annas-archive.li/search?q=Lokayata

I can help with small funds initially for any subscriptions or server setup and etc. as I am tight on time but can afford to fund our works as I mentioned before. I truly wish I had more time to dedicate to the inner details, but hopefully at a later time I could contribute more.

**Comment by einthusan** [comment]
Check out ...

Deep Learning in Indus Valley Script Digitization

Deva Munikanta Reddy Atturu

https://repository.fit.edu/cgi/viewcontent.cgi?article=2460&context=etd

And ...

https://indusscript.in

And ...

![image](https://github.com/user-attachments/assets/9f62f6f0-0a3c-4f99-b04d-38835d695888)

**Comment by ewre324** [comment]
Hi anyone still working on this?
